docs/source/api_reference.rst

```diff
@@ -132,20 +132,3 @@ Widgets can be combined into an interactive dashboard.
 
 .. autoclass:: rubicon_ml.viz.Dashboard
    :members: serve, show
-
-.. _library-reference-workflow:
-
-workflow.prefect
```

Comment:
should've removed this before the last release. needed to change something to test the build